src/index.html

```diff
@@ -1,14 +1,88 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+    href=
+    "https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+    rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+      href="#" class="logo"
+      >
+        <img
+          src="logo.png"
+          alt="Logo Moyo"
+          class="logo__image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link"
+               data-qa="hover"
+               >
```

Comment:
If you have 3 or more attributes in one tag then each of them should be in new line
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="hover"
            >
```